gridstatus/tests/test_ercot.py

```diff
@@ -32,6 +32,14 @@ def test_get_as_prices(self):
         assert df.columns.tolist() == as_cols
         assert df["Time"].unique()[0].date() == date
 
+        date = pd.Timestamp(2022, 11, 8).date()
+        today = utils._handle_date("today").date()
```

Comment:
depending on when the test run, this can give different results for "today". can be fixed with 

```today = utils._handle_date("today", tz=selfiso.default_timezone).date()```

however, looks like `today` is already defined higher up in this test, so not sure this is needed

Reply:
Removed extra implementation of `today` and used original.